src/django/api/views/facility/facility_list_view_set.py

```diff
@@ -414,62 +431,107 @@ def createlist(self, request):
         is_geocoded = False
         parsed_items = set()
         for idx, row in enumerate(rows):
+            # Created a partially filled FacilityListItem for the
+            # create_extendedfields_for_listitem method, as this method
+            # necessitates a row already saved in the database.
             item = FacilityListItem.objects.create(
                     row_index=idx,
                     raw_data=','.join(row.raw_json.values()),
                     raw_json=row.raw_json,
                     raw_header=header_str,
-                    sector=row.sector,
+                    sector=[],
```

Comment:
**Non blocking question:**
Why you change it to empty array when FacilityListItem creating?

Reply:
It can't be empty, but I need to assign some value to it since there are no default values for it.